fenix/app/metrics.yaml

```diff
@@ -10400,3 +10400,305 @@ sync:
       - android-probes@mozilla.com
       - cgordon@mozilla.com
     expires: never
+
```

Comment:
These could really use a tag. See https://searchfox.org/mozilla-central/source/browser/components/shopping/metrics.yaml#10 for an example.

While this would necessitate these moving to their own file, I think it's worth it. Tags really help folks navigate groups of measures in the Glean dictionary.